lib/Service/AvatarService.php

```diff
@@ -89,9 +89,10 @@ private function hasAllowedMime(Avatar $avatar): bool {
 	/**
 	 * @param string $email
 	 * @param string $uid
-	 * @return Avatar|null
+	 * @param bool $cachedOnly
+	 * @return Avatar|null|false
 	 */
-	public function getAvatar(string $email, string $uid): ?Avatar {
+	public function getAvatar(string $email, string $uid, bool $cachedOnly = false): mixed {
```

Comment:
Could we add a new method getCacheAvatar instead of adding a new argument here and changing the return type to mixed?

Reply:
I kind of agree with Daniel. I don't like methods returning either null or false (or an object) because it is hard to understand the semantics without knowing the method's code.